Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -49,6 +49,7 @@ type Props = $ReadOnly<{|
   disabled?: ?boolean,
   focusable?: ?boolean,
   hitSlop?: ?EdgeInsetsProp,
+  id?: ?string,
```

Comment:
Hi @gabrieldonadel, thanks for this PR. However, I don't think that if we are already using an optional property (`id?`) it should also have an optional type `?string`. 

Also, this definition conflicts with some other internal definition we have where the prop is defined as 
```
id?: string
```

Could you update the PR so I can try to reimport it and merge it?

Reply:
Sure, I've just updated it

Reply:
The internal failure was because a component was defining `id` for custom purposes and didn't strip it from the props it forwarded to the RN components. We should check what the type is in React DOM, and patch the internal RN code if  this type should be an optional string...although I don't understand why Flow would complain about receiving a string for an optional string value.